### ISSUE TITLE
Regarding Performance on Imagenet

### ISSUE THREAD

**Issue by mmahesh** [flyyufelix/cnn_finetune#26]
I happened follow the tutorial given in 

https://www.pyimagesearch.com/2016/08/10/imagenet-classification-with-python-and-keras/

But, when i used the pretrained models you provided i happened to get very different results. Please find the script i used [here](https://gist.github.com/mmahesh/9368e3ab1b2febd077aefaede32f1bdd)

When i use your weights the prediction keeps changing everytime i run and for keras default resnet 50 model it shows Magpie as label for the test image ILSVRC2012_test_00000002.JPEG from Imagenet 2012 test data. Is there any problem with the weights? or preprocessing? I think keras default preprocessing step does the same as you mentioned.

Thanks
Mahesh
